### DISCUSSION TITLE
Reload playlist with last position on mix sources

### DISCUSSION THREAD

**Discussion by bugoverfl0w** [savonet/liquidsoap#1774] [Q&A]
Dear team,

I have done my radio stream with mix of multiple sources: micro, playlist, playback (on fallback)

```
channel = 'mychannel'

def apply_metadata(m) =
  print("Now playing: #{m}")
  url = url.encode(m["initial_url"])
  uri = url.encode(m["initial_uri"])
  ignore(http.get("http://localhost/dis.php?f=url=#{url}&uri=#{uri}&channel=#{channel}")) 
end

# Music
security = single("~/tmp/Nguoc_Doi_-_Rhymastic.mp3")

myplaylist = playlist.once("~/tmp/music.m3u", reload_mode = "watch")
radio = myplaylist

radio = fallback(track_sensitive = false, [radio, security])
radio = on_metadata(apply_metadata, radio)

# Stream it out
output.icecast(%vorbis,
  host = "localhost", port = 8000,
  password = "hackme", mount = "basic-radio.mp3",
  radio
  )
```

It works well

But when I add new song to playlist m3u. It reloads, but play on start of playlist, not the last position it lefts

If I use only playlist, no fallback, it works well

I was try to search but there is no result

How can I check this case? 

Thanks,

**Comment by smimram** [comment]
There is no easy way of reloading a playlist while keeping the position: what should we do if the playlist was completely changed? Maybe should you try changing the logic and use a `request.queue` instead of a playlist?
